src/hmrc-design-patterns/page-heading/index.njk

```diff
@@ -68,4 +106,8 @@ layout: twoColumnPage.njk
 
 <p class="govuk-body"><a class="govuk-link" href="https://github.com/hmrc/design-patterns/issues/149">Discuss the page heading on GitHub</a></p>
 
+<h2 class="govuk-heading-l">Changes to this pattern</h2>
+
+<p class="govuk-body">On [date], this pattern replaced <a class="govuk-link" href="/hmrc-design-patterns/hmrc-design-patterns-archive/page-heading/"> an older version</a>. This was because the pattern did not work in all cases, like headings as labels and legends.</p>
```

Comment:
[date] needs to be filled out on date of publishing.

Reply:
Under 'Labels as page headings with subheadings' and 'Legends as page headings with subheadings' should 'H2' be <code>&lt;h2&gt;</code> ?

Under research 'all browsers' isn't really true. Perhaps 'supported browsers'? The grammar in that sentence is awkward too, maybe change to 'The recommended code has been tested with screen reader users, assistive technologies, supported browsers and multiple device types.'

Otherwise all good from me so happy for you to go ahead. I'm happy to update 'what's new' and request the welsh as they're good example tasks for me to work on with Leanne.

Reply:
Thanks Jon. I've updated it.